configure.ac

```diff
@@ -97,6 +97,9 @@ AC_CONFIG_FILES([
     swssconfig/Makefile
     cfgmgr/Makefile
     tests/Makefile
+    utilities/Makefile
```

Comment:
>utilities [](start = 4, length = 9)

We already have sonic-utilities, another `utilities` is confusing.

Reply:
Kind of related to your feedback below, I was trying to group related modules/source files rather than dump them all in the orchagent directory.  Do you (or anyone else reviewing this PR) have any suggestions for a better way of organizing this, or do you think I should just place everything in orchagent?